auto-generated/cluster/stone-prd-rh01/managed/rhtap-releng-tenant/appstudio.redhat.com_v1beta1_remotesecret_pyxis-production-secret-remote-secret.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: appstudio.redhat.com/v1beta1
```

Comment:
actually this secret is probably already defined in the releng workspace...

> oc get remotesecrets
> NAME                                                                AGE
> pyxis-prod-secret-remote-secret                                     125d
> pyxis-staging-secret-remote-secret                                  125d
> 

you can delete this file. 

sorry for the confusion

Reply:
NP at all, fixed!

Thanks for review :)